tensorflow/python/eager/benchmarks_test.py

```diff
@@ -652,6 +674,14 @@ def benchmark_defun_matmul_2_by_2_GPU(self):
       self._benchmark_defun_matmul(
           m, transpose_b=False, num_iters=self._num_iters_2_by_2)
 
+  @test_util.disable_tfrt("With signature")
```

Comment:
@test_util.disable_tfrt("copy to GPU not supported"). this string describes the reason why tfrt is disabled.